cmd/loadbalancer/loadbalancer.go

```diff
@@ -21,26 +21,50 @@ var LoadBalancerCmd = &cobra.Command{
+Each field is separated by ';'.
+If you need multiple instance pools, use the flag multiple times.`
 
 	LoadBalancerCmd.AddCommand(loadBalancerListCmd)
 	LoadBalancerCmd.AddCommand(loadBalancerShowCmd)
-	//loadBalancerCmd.AddCommand(loadBalancerRemoveCmd)
-	//loadBalancerCmd.AddCommand(loadBalancerCreateCmd)
-	//loadBalancerCmd.AddCommand(loadBalancerUpdateCmd)
+	LoadBalancerCmd.AddCommand(loadBalancerRemoveCmd)
+	LoadBalancerCmd.AddCommand(loadBalancerCreateCmd)
+	LoadBalancerCmd.AddCommand(loadBalancerUpdateCmd)
 
 	// Balancer create subcommand
-	// loadBalancerCreateCmd.Flags().StringVarP(&lbName, "name", "", "", "Name of the load balancer")
-	// loadBalancerCreateCmd.Flags().StringVarP(&lbNetwork, "network", "n", "default", "The network to create the loadbalancer")
-	// loadBalancerCreateCmd.Flags().StringVarP(&lbAlgorithm, "algorithm", "a", "", "<round_robin | least_connections> - LoadBalancing algorithm to distribute traffic")
-	// loadBalancerCreateCmd.Flags().StringArrayVarP(&lbBackends, "backends", "b", []string{}, "Specify a backend instance to associate with the load balancer. Takes ip, protocol(optional), source-port, target-port and health-check-port(optional) in the format --backend=ip:instance-ip,protocol:HTTP|TCP,source-port:80,target-port:31579,health-check-port:31580")
-	// loadBalancerCreateCmd.Flags().StringVarP(&lbExternalTrafficPolicy, "external-traffic-policy", "e", "", "optional, Specify the external traffic policy for the load balancer")
-	// loadBalancerCreateCmd.Flags().StringVarP(&lbSessionAffinity, "session-affinity", "s", "", "optional, Specify the session affinity for the load balancer")
-	// loadBalancerCreateCmd.Flags().IntVarP(&lbSessionAffinityConfigTimeout, "session-affinity-config-timeout", "t", 0, "optional, Specify the session affinity config timeout for the load balancer")
-	// loadBalancerCreateCmd.Flags().StringVarP(&lbExistingFirewall, "existing-firewall", "v", "", "optional, ID of existing firewall to use")
-	// loadBalancerCreateCmd.Flags().StringVarP(&lbCreateFirewall, "create-firewall", "c", "", "optional, semicolon-separated list of ports to open - leave blank for default (80;443) or you can use \"all\"")
+	loadBalancerCreateCmd.Flags().StringVarP(&lbName, "name", "", "", "Name of the load balancer")
+	loadBalancerCreateCmd.Flags().StringVarP(&lbNetwork, "network", "n", "default", "The network to create the loadbalancer")
+	loadBalancerCreateCmd.Flags().StringVarP(&lbAlgorithm, "algorithm", "a", "", "<round_robin | least_connections> - LoadBalancing algorithm to distribute traffic")
+	loadBalancerCreateCmd.Flags().StringArrayVarP(&lbBackends, "backends", "b", []string{}, "Specify a backend instance to associate with the load balancer. Takes ip, protocol(optional), source-port, target-port and health-check-port(optional) in the format --backend ip:10.0.0.1|source-port:80|target-port:8080|protocol:TCP|health-check-port:8080")
```

Comment:
the text here is not consistent, `ip:10.0.0.1` should be `ip=10.0.0.1`

Reply:
actually this should be aligned:
	lbBackendExample := `Define backend configurations for the load balancer.
Use this format:
--backend "ip=10.0.0.1|protocol=http|source-port=80|target-port=8080|health-check-port=8080"
Each field is separated by '|'`

there is a whole mismatch